Add whoami endpoint (#57)
cyberark/conjur-openapi-spec: CHANGELOG.md

## [Unreleased]
### Added
- Example use case of spec-generated Ruby client. 

> [cybeark/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)

Reviewer: s/cybeark/cyberark

Also, indent to match the indentation of the line above, starting at the word "Example"

Author: I believe I address these in the /authentication endpoint PR

Reviewer 2: @telday in the `/authentication` endpoint PR you refer here, so neither of them really fix this problem. maybe submit a small PR just to fix this, and then you can rebase everything so they'll all be up to date?

Author: Good point. I didn't see the missing r as well. Just opened a PR

> - Added the /whoami endpoint to the spec file

Reviewer 2: ```suggestion
- The /whoami endpoint is now included in the spec.
```

Please make sure, following our [changelog guidelines](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-guidelines) that the entries are sentences that end in periods and don't repeat the section headers (eg no need to say "Added" again)

We also try to be clear about user impact - it might be nice to say a little about why each endpoint we're documenting is useful in these entries
